DQMOffline/Muon/python/gemEfficiencyHarvester_cfi.py

```diff
@@ -6,9 +6,9 @@
 from DQMOffline.Muon.gemEfficiencyAnalyzer_cfi import gemEfficiencyAnalyzerSta as _gemEfficiencyAnalyzerSta
 
```

Comment:
and the same for `gemEfficiencyHarvester`

Reply:
same with the previous question. I know the configuration files are not currently following the style guide, but I will fix it later in the master branch.

https://github.com/cms-sw/cmssw/blob/c24439ed1375b88eef7173a58c90ff71298547c2/DQMOffline/Muon/src/GEMEfficiencyHarvester.cc#L15-L19